Playground/templates.json

```diff
@@ -16,9 +16,14 @@
     },
     {
         "label": "Load GPU Particle System from Snippet Server",
-        "documentation": "https://doc.babylonjs.com/babylon101/particles#snippet-server",
+        "documentation": "https://doc.babylonjs.com/toolsAndResources/tools/animationCurveEditor#load-from-the-snippet-server",

```

Comment:
is that the right link? it documents loading from the server, but not a particle system